doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -2,112 +2,245 @@
-primarily by resisting against bending. Beams are characterized by
-their second moment of area, their length and their elastic modulus.
-
-In Sympy, we can construct a 2D beam objects with the following properties :
+A planar beam is a structural element that is capable of withstanding load
+through resistance to internal shear and bending. Beams are characterized by
+their length, constraints, cross-sectional second moment of area, and elastic
+modulus. In SymPy, 2D beam objects are constructed by specifying the following
+properties:
 
 - Length
 - Elastic Modulus
 - Second Moment of Area
-- Variable : A symbol that can be used as a variable along the length. By default,
-  this is set to ``Symbol(x)``.
+- Variable : A symbol representing the location along the beam's length. By
+  default, this is set to ``Symbol(x)``.
 - Boundary Conditions
-    - bc_slope : Boundary conditions for slope.
-    - bc_deflection : Boundary conditions for deflection.
+   - bc_slope : Boundary conditions for slope.
+   - bc_deflection : Boundary conditions for deflection.
 - Load Distribution
 
-We have following methods under the beam class:
-
-- apply_load
-- solve_for_reaction_loads
-- shear_force
-- bending_moment
-- slope
-- deflection
+Once the above are specified, the following methods are used to compute useful
+information about the loaded beam:
 
+- ``solve_for_reaction_loads()``
+- ``shear_force()``
+- ``bending_moment()``
+- ``slope()``
 
 Examples
 ========
 
-Let us solve some beam bending problems using this module :
+Below are examples of a variety two dimensional beam bending problems.
 
 Example 1
 ---------
 
-A beam of length 9 meters is having a fixed support at the start.
-A distributed constant load of 8 kN/m is applied downward from the starting
-point till 5 meters away from the start. A clockwise moment of 50 kN-m is
-applied at 5 meters away from the start of the beam. A downward point load
-of 12 kN is applied at the end.
+A cantilever beam 9 meters in length has a distributed constant load of 8 kN/m
+applied downward from the fixed end over a 5 meter distance. A counterclockwise
+moment of 50 kN-m is applied 5 meters from the fixed end. Lastly, a downward
+point load of 12 kN is applied at the free end of the beam.
 
 ::
 
-                                     
-  \\\\|    8 KN/m                    
-  \\\\|_____________                 
-  \\\\|| | | | | | |   ⭯ 50 KN-m     12 KN
-  \\\\|V V V V V V V   |               |
-  \\\\|________________|_______________V
-  \\\\|________________|_______________|
-  \\\\|                                :
-  \\\\|----------------|---------------|
-             5.0 m            4.0 m
+       y
+       ^
+       |
+   \\\\|
+   \\\\|    8 kN/m
+   \\\\|_________________
+   \\\\|| | | | | | | | |             12 kN
+   \\\\|V V V V V V V V V               |
+   \\\\|________________|_______________V
+   \\\\|                |               |
+   \\\\o - - - - - - - -⭯ 50 kN-m - - - | - - -> x
```

Comment:
Worth noting that this ⭯ character doesn't render for me on my phone (it just comes out as a blank space).